kernel/packages/shared/comms/browser.ts

```diff
@@ -0,0 +1,14 @@
+export const compatibleBrowsers: string[] = [
+  "Chrome",
+  "Firefox"
+]
+
+export function isCompatibleBrowser() {
```

Comment:
We can't do allow-lists of browsers, instead I suggest to check the capabilities of the browsers to ensure it will work. I use DCL in brave, and it works perfectly. It also works in latest safari. Edge also works perfectly in guest mode, so as do Opera.

Reply:
Yeah, I agree with you. Maybe we should to re-design the requirements for this issue and not to limit so much our supported browsers, because they are more than we thought it would. And furthermore, fixing the problem that we saw yesterday about the paralel loading of the website and the react app (current cause of the problems we are having with the old versions of Safari), I think we could support a lot of browsers.

I'm going to open the discussion with the team before to continue with the issue 👍 

Thanks for the feedback, Agus.